### ISSUE TITLE
Need Win Install Help

### ISSUE THREAD

**Issue by cheme75** [koush/scrypted#247]
The instructions lack the detail offered for other platforms - win users need some love too. This is as far as I got - firefox error msg, similar failure in edge, Its a real frustration because the docker container looks to be running ok. I just deleted it and will recreate a new yml and try one more time,,,

Secure Connection Failed

An error occurred during a connection to 192.168.1.179:10443. PR_END_OF_FILE_ERROR

    The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the website owners to inform them of this problem.

**Comment by koush** [comment]
I suggest running docker in hyper-v manager.

**Comment by cheme75** [comment]
Any info on how to do that? I can’t seem to find hyper-v manager. I saw it set up 2 virtual adapters - default switch and WSL - both were in 172 subnet while my lan subnet is 192 based so a changed both to my 192 subnet, but that has not done anything to help, at least so far. So I assume I need to somehow bridge the WSL virtual adapter with my physical
LAN adapter, or some other way to get there?

**Comment by cheme75** [comment]
@koush well, I gave up n windows and managed to fumble my way through and got it installed on my old lubuntu machine using docker. It’s picking up my Wyze cams from the Wyze bridge machine and I managed to get the scrypted bridge setup in homekit. Only immediate problem is figuring out how to rotate the doorbell and a v2 that I use sadeways so it views vertically. The homebridge ffmpeg plugin does the rotation perfectly - how do I do same in scrypted?
